includes/validation/class-amp-invalid-url-post-type.php

```diff
@@ -2007,7 +2008,7 @@ public static function get_single_url_page_heading() {
 		}
 
 		/* translators: %s is the name of the page with the the validation error(s) */
-		return esc_html( sprintf( __( 'Errors for: %s', 'amp' ), $name ) );
+		return esc_html( sprintf( __( 'Errors for: %1$s', 'amp' ), $name ) );
```

Comment:
This can remain just `%s` because there is only one placeholder.

Reply:
Oh good catch, I tried adding the icon in here before but realized it was added as text via JS so it didn't work that way.